docs/standard/design-guidelines/choosing-between-class-and-struct.md

```diff
@@ -14,20 +14,21 @@ helpviewer_keywords:
   - "type design guidelines, classes"
 ms.assetid: f8b8ec9b-0ba7-4dea-aadf-a93395cd804f
 ---
-# Choosing Between Class and Struct
-One of the basic design decisions every framework designer faces is whether to design a type as a class (a reference type) or as a struct (a value type). Good understanding of the differences in the behavior of reference types and value types is crucial in making this choice.
+# Choose between class and struct
 
- The first difference between reference types and value types we will consider is that reference types are allocated on the heap and garbage-collected, whereas value types are allocated either on the stack or inline in containing types and deallocated when the stack unwinds or when their containing type gets deallocated. Therefore, allocations and deallocations of value types are in general cheaper than allocations and deallocations of reference types.
```

Comment:
@BillWagner What is the deal with these articles? Since they are part of the design guideline book we're reprinting with permission, are we allowed to update these? I thought we weren't.

Reply:
Will wait on an answer here before merging. I was wondering about this too, although I have updated these files in the past.

Reply:
Consensus is we should not edit anything except metadata.